app/components/shared/supplemental_files_step_body_table_component.html.erb

```diff
@@ -22,5 +22,30 @@
         <% end %>
       <% end %>
     <% end %>
+    <% component.with_row colspan: 5 do |row_component| %>
+      <%= row_component.with_cell do %>
+        <% if with_remove %>
+          <% form_with model: supplemental_file, id: "edit_supplemental_file_#{supplemental_file.id}" do |form| %>
```

Comment:
I wonder if this is "logic-y" enough to want this stuffed in the Ruby side of this component.

Reply:
I'll work through this.